public/app/features/plugins/components/AppRootPage.test.tsx

```diff
@@ -72,13 +73,13 @@ function renderUnderRouter() {
   const route = {
     component: () => <AppRootPage pluginId="my-awesome-plugin" pluginNavSection={appsSection} />,
   } as unknown as RouteDescriptor;
-  locationService.push('/a/my-awesome-plugin');
+  locationService.push(`/a/my-awesome-plugin/${page}`);
 
   render(
     <Router history={locationService.getHistory()}>
       <Provider store={store}>
         <GrafanaContext.Provider value={getGrafanaContextMock()}>
-          <Route path="/a/:pluginId" exact render={(props) => <GrafanaRoute {...props} route={route} />} />
+          <Route path={`/a/:pluginId/${page}`} exact render={(props) => <GrafanaRoute {...props} route={route} />} />
```

Comment:
Is it really necessary to modify an existing test? You are adding a trailing slash here and I wonder if this is somehow changing the way it used to work.

Reply:
We discussed it with @leventebalogh and this was done to not duplicate the code. About slash seems like it does not. Tests are succesful

Reply:
what about `/a/:pluginId/undefined` ?

Reply:
I think Esteban is raising a valid point here, we should probably also cater for the previous scenario, where we are testing if there is no page set ("entry page"). It is a good thing to improve the `renderUnderRouter()` so it can cater for sub-pages, too, but probably in a way where it also serves the previous scenarios.

(Maybe I said something confusing during the pairing, in that case sorry.)